index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Detect JS</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <script src="https://rawgit.com/darcyclarke/Detect.js/master/detect.js"></script>
```

Comment:
In RawGit, they tell us to use cdn endpoint to create cache. But the cache don't expire, so we need to point not to master file, but the release/tag file: `https://cdn.rawgit.com/darcyclarke/Detect.js/v2.2.2/detect.js`